.github/actions/tf-test/action.yml

```diff
@@ -0,0 +1,131 @@
+    default: si.prod.tfe.czi.technology
+    required: false
+  tfe-token:
+    description: An API token to connect to Terraform Enterprise
+    required: true
+  github-token:
+    description: A Github token to use to comment on this PR
+    required: false
+    default: ${{ github.token }}
+  working-dir:
+    description: The current working directory of the TFE workspace
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install SSH key
+      uses: webfactory/ssh-agent@v0.4.1
+      with:
+        ssh-private-key: ${{ inputs.ssh-private-key }}
+    - uses: actions/checkout@v3
+    - name: Config Terraform plugin cache
+      shell: bash
+      run: |
+        echo 'plugin_cache_dir="$HOME/.terraform.d/plugin-cache"' >~/.terraformrc
+        mkdir --parents ~/.terraform.d/plugin-cache
+    - name: Cache Terraform
+      uses: actions/cache@v3
+      with:
+        path: ~/.terraform.d/plugin-cache
+        key: ${{ runner.os }}-terraform-${{ hashFiles('**/.terraform.lock.hcl') }}
+        restore-keys: |
+          ${{ runner.os }}-terraform-
+    - name: Setup Terraform
+      uses: hashicorp/setup-terraform@v2
+      with:
+        terraform_version: ${{ inputs.tf-version }}
+        cli_config_credentials_hostname: ${{ inputs.tfe-hostname }}
+        cli_config_credentials_token: ${{ inputs.tfe-token }}
+    - name: Terraform init
+      id: init 
+      shell: bash
+      working-directory: ${{ inputs.working-dir }}
+      run: terraform init
+    - name: Terraform validate
+      id: validate 
+      shell: bash
+      working-directory: ${{ inputs.working-dir }}
+      run: terraform validate
+    - name: Terraform plan
+      id: plan
+      shell: bash
+      run: terraform plan -no-color
+      working-directory: ${{ inputs.working-dir }}
+    - name: Write plan to PR
+      uses: actions/github-script@v6
+      env:
+        PLAN: "terraform\n${{ steps.plan.outputs.stdout }}"
+      with:
+        github-token: ${{ inputs.github-token }}
+        script: |
+          console.log(context)
+          const prs = await github.rest.repos.listPullRequestsAssociatedWithCommit({
+            context.repository.owner,
+            context.repository.repo,
+            context.sha,
+          });
+          console.log("prs")
+          console.log(prs)
+          // 1. Retrieve existing bot comments for the PR
+          const { data: comments } = await github.rest.issues.listComments({
+            owner: context.repository.owner,
+            repo: context.repository.repo,
+            issue_number: context.issue.number,
+          })
+          const botComment = comments.find(comment => {
+            return comment.user.type === 'Bot' && comment.body.includes('${{ inputs.working-dir }}')
+          })
+
+          // 2. Prepare format of the comment
+          const output = `#### Terraform Format and Style 🖌\`${{ steps.fmt.outcome }}\ \`${{ inputs.working-dir }}\`
```

Comment:
Where does the `fmt` step come from?

Reply:
good catch, this is probably a copy/paste bug